opensearch/src/test/java/org/opensearch/sql/opensearch/data/value/OpenSearchExprValueFactoryTest.java

```diff
@@ -725,24 +726,33 @@ public void constructGeoPoint() {
         constructFromObject("geoV", "42.60355556,-97.25263889"));
   }
 
+  @Test
+  public void constructGeoPointLat() {
+    assertEquals(doubleValue(42.60355556),
+        tupleValue("{\"geoV\":{\"lat\":42.60355556}}").get("geoV").tupleValue().get("lat"));
+    assertEquals(doubleValue(-97.25263889),
+        tupleValue("{\"geoV\":{\"lon\":-97.25263889}}").get("geoV").tupleValue().get("lon"));
+  }
+
```

Comment:
I noticed that this test is called `constrtuctGeoPointLat` but it appears to be testing lat and lon individually, maybe split these tests up into different tests or rename the test to account for this

Reply:
Thanks changed in [920c17c](https://github.com/opensearch-project/sql/pull/1910/commits/920c17c8cf76709a1d4f1b26505ed2c1fd9b2db7)